Dockerfile

```diff
@@ -57,11 +57,11 @@ RUN set -ex; \
     # Clean up
     apk del .build-deps; \
     rm -rf /tmp/pear /var/cache/apk/*
-# DO NOT FORGET TO UPDATE "tags" FILE
-ENV OSTICKET_VERSION=1.15.4 \
-    OSTICKET_SHA256SUM=6c7cf5fe74258dd0beac6176c438468f468a0a3f07f1f6a0349cc68a9e514302
+COPY version-osticket.env /version-osticket.env
 RUN set -ex; \
     \
+    source /version-osticket.env; \
+    rm /version-osticket.env; \
```

Comment:
I don't like this a single bit, as it adds a new, dummy layer, which will still have to be downloaded, extracted, etc. Plus, there might be downstream images which rely on these variables being set. If you wanna set the values externally, build arguments should be used instead and the values should be defined in the compose file. From there, the values can easily be queried, for example, using [yq](https://github.com/kislyuk/yq).

Reply:
As discussed via chat, I've removed this whole change. I'll re-think it and implement it in a future PR so that it isn't a blocker for this one.